packages/localizations/src/en-US.ts

```diff
@@ -664,6 +664,9 @@ export const enUS: LocalizationResource = {
         pwned: 'If you use this password elsewhere, you should change it.',
       },
     },
+    form_param_max_length_exceeded__name: 'Name should not exceed 256 characters.',
```

Comment:
Just double checking - is this the same error FAPI returns?

Reply:
Yes @nikosdouvlis , it's the same error